guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.md

```diff
@@ -0,0 +1,161 @@
+---
+group: javascript-developer-guide
+subgroup: 3_Widgets
+title: Multiselect widget
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+## Overview
+ 
+The Multiselect [widget](https://glossary.magento.com/widget/) is used to create a multiple select with the search field which helps to look for options.
+
+The Multiselect widget source is [lib/web/mage/multiselect.js][].
+
+## Initialize the Multiselect widget
+
+For information about how to initialize a widget in a JS component or `.phtml` template, see the [Initialize JavaScript][] topic.
+
+Generally the Multiselect widget is instantiated like following:
+
+```javascript
+$("#multiselect").multiselect2({});
+```
+
+Where:
+
+- `#multiselect` is the selector of the select element for which Multiselect is initialized.
+
+Phtml template file examples using script:
+
+```html
+<script>
+    require([
+        'jquery',
+        'mage/multiselect'
+    ], function ($) {
+        'use strict';
+        
+        $("#multiselect").multiselect2({});
+    });
+</script>
+```
+
+## Options
+
+The RedirectUrl widget has the following options:
+
+- [mselectContainer](#mselectcontainer)
+- [mselectItemsWrapperClass](#mselectitemswrapperclass)
+- [mselectCheckedClass](#mselectcheckedclass)
+- [containerClass](#containerclass)
+- [searchInputClass](#searchinputclass)
+- [selectedItemsCountClass](#selecteditemscountclass)
+- [currentPage](#currentpage)
+- [lastAppendValue](#lastappendvalue)
+- [updateDelay](#updateDelay)
+
+### `mselectContainer`
+
+Multiselect container selector.
+
+**Type**: String
+
+**Default value**: `'section.mselect-list'`
+
+### `mselectItemsWrapperClass`
+
+Multiselect items wrapper class.
+
+**Type**: String
+
+**Default value**: `'mselect-items-wrapper'`
+
+### `mselectCheckedClass`
+
+Multiselect checked option class.
+
+**Type**: String
+
+**Default value**: `'mselect-checked'`
+
+### `containerClass`
+
+Container class.
+
+**Type**: String
+
+**Default value**: `'paginated'`
+
+### `searchInputClass`
+
+Class of the search input.
+
+**Type**: String
+
+**Default value**: `'admin__action-multiselect-search'`
+
+### `selectedItemsCountClass`
+
+Class of the selected items counter.
+
+**Type**: String
+
+**Default value**: `'admin__action-multiselect-search'`
+
+### `currentPage`
+
+Current page of options.
```

Comment:
It would be great to extend this explanation as well since it's not obvious what options are we talking about. 
If we are talking about multi-select items, it's a good idea to name "options" as "multi-select items" because the term "options" in case of explaining the code implementation is ambiguous.
This suggesting is fair for the explanations below as well. 

Thank you